native/src/binaryRead.cpp

```diff
@@ -1947,6 +1941,11 @@ void loadTransportRoutes(BinaryMapFile* file, vector<int32_t> filePointers, UNOR
 				finishInit.push_back(transportRoute);
 			}
 		}
+		lseek(file->routefd, 0, SEEK_SET);
+		FileInputStream input(file->routefd);
+		input.SetCloseOnDelete(false);
+		CodedInputStream cis(&input);
+		cis.SetTotalBytesLimit(INT_MAX, INT_MAX >> 1);
```

Comment:
Hey. Thanks for fixing. Just curious: and what about tests? Do you implement them? If not then why? Thnx!